js/client-chat.js

```diff
@@ -601,9 +601,29 @@
 				} else {
 					app.ignore[toUserid(target)] = 1;
 					this.add("User '" + toName(target) + "' ignored. (Moderator messages will not be ignored.)");
+					app.saveIgnore();
 				}
 				return false;
 
+			case 'clearignore':
+				if (this.checkBroadcast(cmd, text)) return false;
+				if (!target) {
+					this.parseCommand('/help ignore');
+					return false;
+				}
+				if (toID(target) !== 'confirm') {
+					this.add("Are you sure you want to clear your ignore list?");
+					this.add("Type /clearignore confirm if you're sure.");
```

Comment:
The boundary between `confirm` and `if` is very unclear.

Ideally we should use code blocks, but I don't actually know how to put code blocks into `.add`

Reply:
```suggestion
					this.add('|html|If you\'re sure, use <code>/clearignore confirm</code>');
```